docs/src/basic_example.md

```diff
@@ -0,0 +1,197 @@
+include third-order mixed derivatives. In primitive variables ``q = (\eta, v)`` they can be written as:
+
+```math
+\begin{aligned}
+  \eta_t + ((\eta + D)v)_x - \frac{1}{6}(D^2\eta_{xt})_x &= 0,\\
+  v_t + g\eta_x + \left(\frac{1}{2}v^2\right)_x - \frac{1}{6}(D^2v_t)_{xx} &= 0.
+\end{aligned}
+```
+
+All equations in DispersiveShallowWater.jl follow the same variable conventions: ``\eta = h + b`` describes the total water height, ``h`` the water height above the bottom topography (bathymetry), ``b = \eta_0 - D`` the bathymetry, and ``v`` the velocity in horizontal direction. The reference water height ``\eta_0`` (also called still water height) and gravitational acceleration ``g`` are used consistently across all equations. For the BBM-BBM equations specifically, ``\eta_0`` is typically set to 0.
+
+A sketch of the water height and bathymetry can be found below.
+
+![water height and bathymetry](bathymetry.png)
+
+## [Getting started](@id basic_example)
+
+In order to conduct a numerical simulation with DispersiveShallowWater.jl, we perform the following steps.
+
+First, we load the necessary libraries:
+
+```@example overview
+using DispersiveShallowWater, OrdinaryDiffEqTsit5
+```
+
+## Define physical setup
+
+As a first step of a numerical simulation, we define the physical setup we want to solve. This includes the set of equations, potentially
+including physical parameters, initial and boundary conditions as well as the domain. In the following example, the initial condition
+describes a traveling wave that moves towards a beach, which is modeled by a linearly increasing bathymetry.
+
+```@example overview
+equations = BBMBBMEquations1D(bathymetry_type = bathymetry_variable, gravity = 9.81)
+
+function initial_condition_shoaling(x, t, equations::BBMBBMEquations1D, mesh)
+    A = 0.07 # amplitude of wave
+    x0 = -30 # initial center
+    eta = A * exp(-0.1*(x - x0)^2)
+    v = 0
+    D = x <= 0.0 ? 0.7 : 0.7 - 1/50 * x
+    return SVector(eta, v, D)
+end
+
+initial_condition = initial_condition_shoaling
+boundary_conditions = boundary_condition_periodic
+
+coordinates_min = -130.0
+coordinates_max = 20.0
+N = 512
+mesh = Mesh1D(coordinates_min, coordinates_max, N)
+```
+
+The first line specifies that we want to solve the BBM-BBM equations with variable bathymetry using a gravitational acceleration of ``g = 9.81``.
+Afterwards, we define the initial condition, which is described as a function with the spatial variable `x`, the time `t`, the `equations`, and
+a `mesh` as parameters.
+
+If an analytical solution is available, the time variable `t` can be used, and the initial condition can serve as an analytical solution to be compared with the numerical solution. Otherwise, you can just keep the time variable unused.
+
+An initial condition in DispersiveShallowWater.jl is supposed to return an `SVector` holding the values for each of the unknown variables. Since the bathymetry is
```

Comment:
```suggestion
An initial condition in [DispersiveShallowWater.jl](https://github.com/NumericalMathematics/DispersiveShallowWater.jl) is supposed to return an `SVector` holding the values for each of the unknown variables. Since the bathymetry is
```